.ldrelease/build-docs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+PROJECT_DIR=$(pwd)
+
+doxygen
+
+cp -r "${PROJECT_DIR}"/docs/build/html/* "${LD_RELEASE_DOCS_DIR}"
```

Comment:
I also removed the comment which is no longer relevant, and quoted the env variables.